### ISSUE TITLE
"calibrated using readings from last 4 days" (or words to that effect error) 

### ISSUE THREAD

**Issue by Son1k02** [NightscoutFoundation/xDrip#473]
Hi guys, xdrip is reading 17.8mmol/L & has been in for just over 4 days. The last 2 days I've had readings higher that I've had using the libre & blucon. I am away from home atm & didn't calibrate yesterday, only buying strips this morning. I've just tested my bsl & got a reading of 11.8mmol/L (after diner, so not unusual that it was out of target). I've tried to calibrate xdrip, but an error comes up, essentially saying that it is ignoring my calibration as it likes the data from a few days ago better.

I have always entered calibrations & had the data adjust to the calibration. Is there a new setting that I'm unaware of that has caused this change, or was calibrating it while high the day before the cause of this problem. I'd like to fix this issue asap, I'd be thankful for a solution, or I'll just try resting the calibrations in a few hours. Thanks in advance!

*"calibrated using readings from beyond last 4 days"

**Comment by jamorham** [comment]
It just means its using more than 4 days worth of data as it doesn't have so many calibrations from the last few days. The calibration graph can show you the calibrations being used.

**Comment by petedcurtis** [comment]
Hi, First I'd like to thank the developers of xDrip app for making it available.  I just got a Abbott Freestyle Libre Meter & 3 sensors (1 month supply) about 2 weeks ago subsidized by Medicare. Bad news is my first sensor dropped off after 7 days after I got wet in a rain storm (OK I'll use some Micropore tape going forward), and the second one I dropped on the floor when replacing the first on my other arm, so I'm on the last one for the month.  So I'll have to go out and buy some replacements at full cost from my Walmart Pharmacy.  Even so I like Libre system very much but its calibration seemed to vary away to much from my OneTouch Ultra Test Strip readings, more so on my second sensor and being a NFC flash reader no real time low sugar alerts. So I saw the Ambrosia NightRider NFC to BT adapter and purchased one. Kudos to Ambrosia for the neat piece of  hardware at an affordable price. I downloaded the Blu Con app and found it was very basic: It didn't seem to re-calibrate to my Test strip and so calibration was up to 100 mg/Dl different, also it was difficult for it to scan and find the NightRider BT. So I found xDrip+ app and just started using it.  So far it seems to calibrate to my test strips.     BTW: I'm a 70 Yr old Brit retired ex Electronics guy diagnosed 20 yrs ago with Type 2 diabetes living in FL, USA since 1998. I'm very insulin resistant and use 200 units a day + via a Medtronic 722 Pump, last A1c was 7.2.
